pvr.vuplus/addon.xml.in

```diff
@@ -175,6 +175,9 @@
     <disclaimer lang="zh_TW">這是測試版軟體！其原創作者並無法對於以下情況負責，包含：錄影失敗，不正確的定時設定，多餘時數，或任何產生的其它不良影響...</disclaimer>
     <platform>@PLATFORM@</platform>
     <news>
+v4.5.1
+- Update: Kodi VFS API 1.0.3
```

Comment:
By changed this then a bit bigger?

Reply:
Huh?

Reply:
I think the PR is fine as it is.

Reply:
Like this https://github.com/kodi-pvr/pvr.vuplus/pull/242/commits/32d4ba336a65cbfbdbcc7d6c4e25ce5e0888b24b#diff-3a5a56a800c8b8c56b918f4ddf66f45dR178-R182 ?

Reply:
OK :smile:

Reply:
Ah, yes, that would be good ;) Please do add those!

Reply:
Also needs to be added in changelog.txt

Reply:
Is added on both